lib/backend/report.go

```diff
@@ -408,8 +408,8 @@ func buildKeyLabel(key string, sensitivePrefixes []string) string {
 // sensitive values.
 var sensitiveBackendPrefixes = []string{
 	"tokens",
-	"resetpasswordtokens",
-	"adduseru2fchallenges",
+	"usertoken",
+	"sessionData",
```

Comment:
i did attempt to make this a tad more resilient by trying to import the keys, but it created a import cycle problem (`lib/local` imports from `lib/backend` and `lib/backend` imports from `lib/local`), so I left it as is.